scripts/clan/starclan.py

```diff
@@ -0,0 +1,40 @@
+import pygame
+
+from scripts.cat.sprites import sprites
+
+
+class StarClan:
```

Comment:
Could we rename this to DeadClan?  Since this class handles all the dead cats, not just StarClan (or, at least, i presume it does, since all dead cats fade regardless of group), it seems like that would make more sense.

Reply:
I was debating doing that but then I hesitated 😂 I'll make that change.